src/main/java/com/cburch/logisim/tools/MatrixPlacerDialog.java

```diff
@@ -57,7 +57,7 @@ public MatrixPlacerDialog(MatrixPlacerInfo value, String name, boolean AutoLable
     c.anchor = GridBagConstraints.CENTER;
     c.weightx = 1;
     c.weighty = 1;
-    if ((MatrixInfo.getLabel() != null) & AutoLablerActive) {
+    if ((MatrixInfo.getLabel() != null) && AutoLablerActive) {
```

Comment:
Typo: `AutoLablerActive` -> `AutoLabelerActive`

Maybe, there are other uses of "labler" instead of "labeler" in the code base.

Reply:
I believe it is just a typo. Will fix later.

Reply:
@MarcinOrlowski: I agree it is a typo and just wanted to draw your attention to it.